CHANGELOG.md

```diff
@@ -5,12 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.21.0] - 2021-07-30
 
 ### Features
 
-- Method to initiate an MFA step-up challenge
-- Method to list registered MFA credentials
+- Initiate an MFA step-up challenge
```

Comment:
It would be better if there were links to the documentation of the related methods.

Reply:
I didn't find any documentation on the website :/ will add it later yes